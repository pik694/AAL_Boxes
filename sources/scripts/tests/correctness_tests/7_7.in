7
94 76 15
21 17 94
40 7 72
48 60 60
57 33 57
55 37 33
51 8 46

