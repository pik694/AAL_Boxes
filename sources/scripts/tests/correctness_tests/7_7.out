6 4 3 2 0 1 5 
526818->380673
